docs/reference/cli.rst

```diff
@@ -173,3 +173,15 @@ Example::
 
     This command publishes drafts. You should review drafts before using this
     command, because they will become public.
+
+**********************
+Maintenance and repair
+**********************
+
+.. _fix-mptt:
+
+``fix-mptt``
```

Comment:
Possibly not, but since we tell people to use it in the release docs, it had better be explained!